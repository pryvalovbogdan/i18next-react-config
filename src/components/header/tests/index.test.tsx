import { renderWithProviders } from '../../../utils/testsUtils.tsx';

import { Header } from '../index.tsx';

describe('Test Header', () => {
  test('snapshot', () => {
    const { baseElement } = renderWithProviders(<Header />);

    expect(baseElement).toMatchSnapshot();
  });
});
