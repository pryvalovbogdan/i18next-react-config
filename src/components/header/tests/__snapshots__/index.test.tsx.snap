// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header snapshot 1`] = `
<body>
  <div />
</body>
`;
